lib/emergency_banner/display.rb

```diff
@@ -0,0 +1,34 @@
+module EmergencyBanner
+  class Display
+    def initialize
+      @redis = Redis.new
```

Comment:
If we are doing a refactor, we might as well clean this up a bit?
There's no real reason to expose `@redis` unless we want to inject it in when initialising the class.

If we don't care about that then we could hide it under `private`:

```
private
  def connect
    Redis.new
  end

  def content
    @data ||= connection.hgetall("emergency_banner")
    ...
  end
```